src/relayer/RelayerConfig.ts

```diff
@@ -61,11 +67,11 @@ export class RelayerConfig extends CommonConfig {
     this.relayerDestinationChains = JSON.parse(RELAYER_DESTINATION_CHAINS ?? "[]");
 
     // Empty means all tokens.
-    this.relayerTokens = RELAYER_TOKENS
-      ? JSON.parse(RELAYER_TOKENS).map((token) => ethers.utils.getAddress(token))
-      : [];
+    this.relayerTokens = RELAYER_TOKENS ? JSON.parse(RELAYER_TOKENS).map((token: string) => getAddress(token)) : [];
```

Comment:
Nit: follows the below changes
```suggestion
    this.relayerTokens = JSON.parse(RELAYER_TOKENS || "[]").map((token: string) => getAddress(token));
```